custom/benin/tasks.py

```diff
@@ -0,0 +1,8 @@
+from corehq.apps.celery import task
+
+
+@task(queue='background_queue', ignore_result=True)
+def process_updates_for_village_async(domain, village, dry_run):
```

Comment:
celery is great approach. Just one thing to note is that once things are in celery,  you won't have any visibility/control on the updates as you lose the logging and once the tasks are queued up you don't know when they would get picked up.

Just good to consider that before you finalize the approach.

Reply:
Noted. That is a valid concern. 
I have added a new logger (file) for the script however I am not sure if it work when tasks are run across multiple celery machines.